README.md

```diff
@@ -59,6 +59,33 @@ work on OS X if the databases are all installed using
 To run the linter without running any tests, run `npm run lint`. To
 automatically fix linting errors run `npm run lint-fix`.
 
+### Using Docker for Testing
+
+Running the testsuite on _Jenkins_  is based on docker images. You can also make use of this setup when running tests locally. Scripts are provided for different stages of testing: testing the documentation, running tests against different `nodejs` versions and running tests against different versions of dependencies.
```

Comment:
We should spell it "Node.js" instead of "nodejs" (also remove the surrounding "\`" as I don't think the reference to the `nodejs-version` command argument is important here).